### PR TITLE
Yaml file for GHCR 

### PR DESCRIPTION
**Pull request by HauwaHakimi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#RUN chown -R user:user /opt/app
+WORKDIR /opt/app
+
+COPY . .
+
+RUN bundle install
+RUN npm install
+
+# Expose port 8888
+EXPOSE 8888
```
Comment (tabroughton) [inline Dockerfile:20]: pebble: is there a reason we aren't using the default port 3000?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,6 @@ This is continuously deployed from master [by Github Actions](https://github.com
 
 To deploy to a separate route (e.g. for testing / previewing changes) you can push to PaaS manually:
 
-```
+```sh
```
Comment (tabroughton) [inline README.md:41]: pebble: do we need new details in the readme to describe the dockerfile and how to build/use it locally?

File: Gemfile.lock [hunk 3]
```diff
@@ -94,7 +94,7 @@ GEM
     listen (3.8.0)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    lograge (0.12.0)
+    lograge (0.14.0)
```
Comment (tabroughton) [inline Gemfile.lock:97]: grit: are all the dependencies updated/patched to address any security concerns etc or are they to get the devenv up and running only atm?  (will we patch/address dependabot issues in another ticket?).

**Review by tabroughton — CHANGES_REQUESTED** [review]
just a few questions from me / points to address
I use: grit, gravel, pebble, stone, boulder, mountain (as a sizing of imporatance) to respond/address

File: Dockerfile (hunk 1 above)
Comment (HauwaHakimi) [reply Dockerfile:20]: @tabroughton Initially, we had set it to port 3000, but encountered issues accessing the docker container locally during our tests, which led us to switch to port 8888. Interestingly, it eventually worked on port 3000, but I opted to keep it at 8888 at that moment. If there's a preference or a specific reason to revert back to port 3000, I'm more than willing to make that change.

File: README.md (hunk 2 above)
Comment (owenblacker) [reply README.md:41]: Yes; that's still marked as to-do in [ENG-83](https://gds-cyber-and-engineering.atlassian.net/browse/ENG-83)

File: Gemfile.lock (hunk 3 above)
Comment (owenblacker) [reply Gemfile.lock:97]: The latter. Updating dependencies is separate in the backlog

File: Dockerfile (hunk 1 above)
Comment (owenblacker) [reply Dockerfile]: This has since been changed back to 3000

**Pull request closed without merging**